### DISCUSSION TITLE
The static image loads twice. What's the reason

### DISCUSSION THREAD

**Discussion by 3282778869** [nuxt/nuxt#25582] [Questions]
Each time you enter the page, the static image in the page will load twice. After looking at the generated html, it will generate a link tag
![image](https://github.com/nuxt/nuxt/assets/113400444/7ce6d547-e55d-4cf5-bd8b-4ab53cb9701b)

**Comment by undermoonn** [comment]
Link tag 是 `preload` 或者 `prefetch`  吗

**Reply by 3282778869** [reply]
是 prefetch

**Reply by undermoonn** [reply]
<img width="436" alt="截屏2024-02-04 12 45 45" src="https://github.com/nuxt/nuxt/assets/39266501/5d731fc6-1a89-4800-88dd-d291cfda7a51">
所以是有两个相同的 link prefetch 地址吗，可以贴一下 HTML

**Reply by 3282778869** [reply]
![image](https://github.com/nuxt/nuxt/assets/113400444/7d903505-f4d4-4d54-ad5f-0a5c6f6d49ea)
是我使用img标签引入这张图片的时候 ，打包以后去运行的时候，就会给我生成一个  link 引入图片的标签，然后link引入的那个图片是prefetch 的方式，

**Reply by undermoonn** [reply]
Nuxt 本身没有 prefetch 图片的行为。可以看看是不是哪个 module 的行为，或者是手动调用 useHead 写的 prefetch 呀

**Reply by 3282778869** [reply]
里面没有引入任何的第三方包，就只有一个pinia的包 然后什么都没写 直接打包以后放到线上就变成这样啦 让人特别的蒙
